### DISCUSSION TITLE
how can i change the size of the button?

### DISCUSSION THREAD

**Discussion by poppintiger** [zauberzeug/nicegui#1361] [Q&A] [answered]
### Description

I have 2 colums of elements, the button fill full of the grid, how can I change the size of the button?
```py
with ui.grid(columns=2):   
            ui.textarea(label='Text', placeholder='start typing')
            ui.button('chat', on_click=lambda: ui.open('documentation#open')).style("size=10px")
```
![image](https://github.com/zauberzeug/nicegui/assets/26161307/48549ead-2475-463d-b8c3-f560193ad050)

**Comment by falkoschindler** [comment, marked as answer]
The default behavior of grid cells is to stretch across the available space. That's why the button fills the whole height.

You can avoid that using the "items-start" class:
```py
with ui.grid(columns=2).classes('items-start'):
    ui.textarea(label='Text', placeholder='start typing')
    ui.button('chat', on_click=lambda: ui.open('documentation#open'))
```
Here is a detailed explanation of all options: https://tailwindcss.com/docs/align-items.

**Reply by poppintiger** [reply]
Thanks so much!!!
